extensions/opentelemetry/opentelemetry/deployment/src/main/java/io/quarkus/opentelemetry/deployment/OpenTelemetryProcessor.java

```diff
@@ -128,4 +185,11 @@ void storeVertxOnContextStorage(OpenTelemetryRecorder recorder, CoreVertxBuildIt
     public static boolean isClassPresent(String classname) {
         return QuarkusClassLoader.isClassPresentAtRuntime(classname);
     }
+
+    static class WithSpanPredicate implements Predicate<AnnotationInstance> {
```

Comment:
This class seems redundant and the predicate uses volkswagen logic anyway :)

Reply:
lol. You are right... Was a forgotten leftover.

Reply:
Correct. It was forgotten. Dropped it.